src/software/ai/navigator/path_manager/obstacle_path_manager.cpp

```diff
@@ -0,0 +1,79 @@
+#include "software/ai/navigator/path_manager/obstacle_path_manager.h"
+
+/**
+ * This file contains the implementation of the obstacle path manager
+ * which uses obstacles to arbitrate between paths
+ * ret a path consisting of only the start and destination
+ * points.
+ */
+
+
+ObstaclePathManager::ObstaclePathManager(std::unique_ptr<PathPlanner> path_planner)
+    : path_planner(std::move(path_planner))
+{
+}
+
+const std::map<int, Path> ObstaclePathManager::getManagedPaths(
+    const std::map<int, PathObjective> &objectives, const Rectangle &navigable_area,
+    const std::vector<Obstacle> &static_obstacles)
+{
+    std::map<int, Path> retval;
```

Comment:
perhaps `managed_paths` might be a bit more descriptive?

Reply:
done